docs/PacketFence_Installation_Guide.asciidoc

```diff
@@ -666,15 +669,15 @@ From the 'Configuration -> Policies and Access Control -> Authentication Sources
 * *Bind DN:* CN=Administrator,CN=Users,DC=acme,DC=local
 * *Password:* acme123
 
-Then, we add an 'Authentication rules' by clicking on the 'Add rule' button and provide the following information:
+Then, we add an 'Authentication rules' by clicking on the `Add rule` button and provide the following information:
 
 [options="compact"]
 * *Name:* employees
 * *Description:* Rule for all employees
 * Don't set any condition (as it's a catch-all rule)
 * Set the following *actions:*
-  - Set role employee
-  - Set unregistration date January 1st, 2020
+  * Role - employee
+  * Unregistration date - January 1st, 2020
```

Comment:
I will make more sense to use Access duration here in place of using Unregistration date with a wrong value.